src/layout.rs

```diff
@@ -284,6 +292,8 @@ pub struct Layout {
     /// Whether the last chunk of the computed layout should be expanded to fill the available
     /// space.
     expand_to_fill: bool,
+    /// Whether each chunk should be similar in size to every other chunk
+    chunk_size_preference: ChunkSizePreference,
```

Comment:
I would have liked to do this:

```
            chunk_size_preference: Default::default(),
```

But I can't call `non-const` in `const` functions.

Reply:
Yeah - it's really annoying that trait methods like `Default::default()` can't be const :/ There's an RFC to fix it though - so someday...